README.md

```diff
@@ -219,15 +219,16 @@ An example of using `fastify-swagger` with `static` mode enabled can be found [h
  | exposeRoute        | false            | Exposes documentation route.                                                                                              |
  | hiddenTag          | X-HIDDEN         | Tag to control hiding of routes.                                                                                          |
  | hideUntagged       | false            | If `true` remove routes without tags from resulting Swagger/OpenAPI schema file.                                          |
+ | hideRoutePredicate | null             | predicate function (url, schema) => boolean to decide if should hide route                                                |
```

Comment:
name suggestions are welcomed :)

Reply:
I'm not really convinced by the name, why "predicate"?

Reply:
Because it's common enough to use `predicate` when refering to a function that returns true or false.
here's a result from google with more details:

https://www.baeldung.com/cs/predicates

you're more than welcome to suggest any other name, i'm open to changes!

I also considered just calling it `hideRouteFn` but from my own experience shortening function to 'Fn' is not very welcome on libraries.